### DISCUSSION TITLE
Kick anti-cheat

### DISCUSSION THREAD

**Discussion by ILaptev96** [PrismarineJS/mineflayer#2694] [Q&A]
Hi. My bot is getting kicked by the anti-cheat.
I made anti-afk bot, he walks, twirls his head and jumps, I noticed the slower he is, the less likely to poo the anti-cheat.
I made it as slow as possible and similar to the player's behavior, but it still gets kicked about once every 5 hours, what can you do to avoid it?

**Comment by amoraschi** [comment]
Make it reconnect when it gets kicked

**Reply by ILaptev96** [reply]
I need to bypass the system so that it doesn't kick at all, otherwise I get banned
